pollination/sky_irradiance/entry.py

```diff
@@ -171,28 +190,67 @@ def create_sky(
             }
         ]
 
+    @task(
+        template=SplitGridFolder, needs=[create_rad_folder],
+        sub_paths={'input_folder': 'grid'}
+    )
+    def split_grid_folder(
+        self, input_folder=create_rad_folder._outputs.model_folder,
+        cpu_count=cpu_count, cpus_per_grid=3, min_sensor_count=min_sensor_count
```

Comment:
In this case, we only have one simulation for the total sky. `cpus_per_grid` should be set to 1.

```suggestion
        cpu_count=cpu_count, cpus_per_grid=1, min_sensor_count=min_sensor_count
```